src/app/survey/survey.component.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+<<<<<<< HEAD
```

Comment:
merge conflicts in this file, but this might should be completely removed anyway.  Make the changes here (if any) to `app/review.review.component.spec.ts`